requirements.txt

```diff
@@ -4,6 +4,6 @@ requests == 2.24.0
 shapely == 1.7.1
 tensorflow == 2.2.0
 tqdm == 4.45.0
-numpy == 1.17.4
-pyyaml == 5.4
+numpy == 1.17.3
+pyyaml == 5.3.1
 scipy
```

Comment:
tensorflow >= 2.2
numpy >= 1.17.3

would be more consistent

Reply:
ok, to note that pip will then install the latest version of the packages.
Tensorflow 2.6.0
numpy 1.19.5
pyyaml 5.4.1